xla/hlo/ir/hlo_computation.cc

```diff
@@ -343,22 +343,21 @@ void HloComputation::RemoveCallee(const HloInstruction* caller,
   }
 }
 
-absl::flat_hash_map<const HloInstruction*, int>*
-HloComputation::GetCallersMap() {
+absl::flat_hash_map<HloInstruction*, int>* HloComputation::GetCallersMap() {
   if (static_cast<CallersType>(callers_ & kCallerTypeMask) ==
       CallersType::kCallerCountHashMap) {
-    return reinterpret_cast<absl::flat_hash_map<const HloInstruction*, int>*>(
+    return reinterpret_cast<absl::flat_hash_map< HloInstruction*, int>*>(
         callers_ & ~kCallerTypeMask);
   }
   return nullptr;
 }
 
-absl::flat_hash_map<const HloInstruction*, int>* const
-HloComputation::GetCallersMap() const {
+absl::flat_hash_map<HloInstruction*, int>* const HloComputation::GetCallersMap()
+    const {
   if (static_cast<CallersType>(callers_ & kCallerTypeMask) ==
       CallersType::kCallerCountHashMap) {
     return reinterpret_cast<
-        absl::flat_hash_map<const HloInstruction*, int>* const>(
+        absl::flat_hash_map< HloInstruction*, int>* const>(
```

Comment:
nit: there is one extra space before HloInstruction*, but possibly this will be fixed automatically when importing this PR. Will let you know in case it doesn't get automatically fixed.

Reply:
Yes, it's fixed.